.env

```diff
@@ -2,14 +2,14 @@
 VERBOSE_HARNESS=0
 
 # Used to determine sandbox build type:
-TYPE="channel" # OR "source"
+TYPE="source" # OR "source"
 
 # Used when TYPE==channel:
 ALGOD_CHANNEL="nightly"
 
 # Used when TYPE==source:
-ALGOD_URL="https://github.com/algorand/go-algorand"
-ALGOD_BRANCH="master"
+ALGOD_URL="https://github.com/jasonpaulos/go-algorand"
```

Comment:
TODO: Set this back to master when experimental tags are removed from simulate endpoint